scripts/docker-compose-openfn.yml

```diff
@@ -5,16 +5,16 @@ networks:
 services:
   postgresql:
     env_file:
-      - '${MSF_ENV_CONFIG_PATH}/.env'
-    stop_grace_period: '3s'
+      - "${MSF_ENV_CONFIG_PATH}/.env"
```

Comment:
Are these changes necessary for this PR? If not, please revert the changes to this file